MM2RandoLib/Resources/Asm/Options/self_destruct.asm

```diff
@@ -0,0 +1,20 @@
+.include "mm2r.inc"
+
+.segment "BANKE"
+
+.org $817e ; Part of main game loop
+	jsr PatchDoPauseScreen ; Was jsr DoPauseScreen
+
+.reloc
+PatchDoPauseScreen:
+	; Okay to clobber A, X, Y?
```

Comment:
I don't particularly like to see questions in the comments lol. Why is it okay to clobber these registers?

Reply:
A and X it's safe to clobber, as they're not used again until they're loaded. Y I'm not 100% sure of, as it's not clear when the next use is.